src/Item/PluginItem/ItemFooter.js

```diff
@@ -54,6 +54,7 @@ class ItemFooter extends Component {
                     <ItemDescription description={description} />
                     <hr style={style.line} />
                     <Interpretations
+                        object={this.props.item}
```

Comment:
What about just sending object, and not objectType and objectId, since Interpretations can derive those values from object? Seems like a lot of noise to break apart object and send all its different properties separately

Reply:
Removed objectType, but kept objectId to avoid having to call getId everywhere.